src/pspm_simple_qa.m

```diff
@@ -183,7 +183,7 @@
 if isfield(options, 'missing_epochs_filename')
     if ~isempty(find(filt == 0, 1))
         epochs = filter_to_epochs(filt);
-		epochs = epochs / sr; %convert into integers
+		epochs = epochs / sr; %convert into seconds
```

Comment:
Sorry I added wrong comments... I was thinking about converting it into integers and then reminded we shall use the direct output without rounding, but I forgot to change the annotation, my fault..